### ISSUE TITLE
Add loading bar when generating project

### ISSUE THREAD

**Issue by retzd-tech** [MicrofrontendGenerator/generator#5]

**Comment by ProCode2** [comment]
Hey @retzd-tech , can you elaborate on this issue? And please assign me. 🙌

**Comment by retzd-tech** [comment]
> Hey @retzd-tech , can you elaborate on this issue? And please assign me. 🙌

Yes, basically, when generating the project files there is no loading bar, it just a console log of "Generating ...."

The expectation is

There is a loading bar from left to right when generating project until finished, especially when executing the sh command
